src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -87,6 +96,186 @@ private void RemoveOperations(CodeModel codeModel)
             }
         }
 
+        private void OmitOperationGroups(CodeModel codeModel, BuildContext<MgmtOutputLibrary> context)
+        {
+            var operationGroupsToOmit = context.Configuration.MgmtConfiguration.OperationGroupsToOmit;
+            if (operationGroupsToOmit != null)
+            {
+                var omitSet = operationGroupsToOmit.ToHashSet();
+                for (int i = 0; i < codeModel.OperationGroups.Count; i++)
```

Comment:
nit: foreach

Reply:
Not allowed to use foreach, need to update codeModel.OperationGroups in this loop